tests/completion/indirect/task.yaml

```diff
@@ -12,16 +12,17 @@ prepare: |
 
 restore: |
   (
-      cd ../../lib/snaps/test-snapd-complexion
+      cd ../../lib/snaps/test-snapd-complexion || exit 1
       mv test-snapd-complexion.bash-completer.orig test-snapd-complexion.bash-completer
       snap remove test-snapd-complexion
   )
 
 execute: |
   d="$PWD"
+  #shellcheck disable=SC1090
   source "${SPREAD_PATH}/${SPREAD_SUITE}/${SPREAD_VARIANT}.vars"
```

Comment:
Can we just use . instead of source here?

Reply:
After reading https://github.com/koalaman/shellcheck/wiki/SC1090 I think this is fine.